microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -497,7 +508,53 @@ def _configure_vdpa_devices(self, node: Node) -> None:
             node.tools[Chown].change_owner(file=PurePath(device_path), user=user)
             node.tools[Chmod].chmod(path=device_path, permission=permission, sudo=True)
 
-    def _set_data_disk(self) -> None:
+    def _get_pmem_for_block_tests(self) -> str:
+        """
+        Following is the last usable entry in e829 table and is safest to use for
+        pmem since its most likely to be free. 0x0000001000000000 is 64G.
+
+        [    0.000000] BIOS-e820: [mem 0x0000001000000000-0x00000040ffffffff] usable
+
+        """
+        memmap_str = "memmap=16G!64G"
```

Comment:
Better to make this configurable from runbook. So that we can easily run these tests on different servers.

Reply:
Made it configurable.